src/test/java/org/jenkinsci/plugins/workflow/cps/CpsFlowDefinitionTest.java

```diff
@@ -0,0 +1,43 @@
+package org.jenkinsci.plugins.workflow.cps;
+
+import org.junit.Test;
+
+/**
+ * @author Kohsuke Kawaguchi
+ */
+public class CpsFlowDefinitionTest extends AbstractCpsFlowTest {
+    /**
+     * CpsFlowDefinition's simplest possible test.
+     */
+    @Test
+    public void simplestPossibleTest() throws Exception {
+        CpsFlowDefinition flow = new CpsFlowDefinition(
+                "def sqrt(int x) { return Math.sqrt(x); }\nfor (int i=0; i<10; i++)\nsqrt(i);",
+                false);
+
+        createExecution(flow);
+        exec.start();
+        exec.waitForSuspension();
+
+        assert exec.isComplete();
+    }
+
+    @Test
+    public void exceptionInWorkflowShouldBreakFlowExecution() throws Exception {
+        CpsFlowDefinition flow = new CpsFlowDefinition(
+                "throw new Throwable('This is a fire drill, not a real fire');",
+                false);
+
+        // get this going...
+        createExecution(flow);
+        exec.start();
+
+        // it should stop at watch and suspend.
+        exec.waitForSuspension();
+        assert exec.isComplete();
+        Throwable t = exec.getCauseOfFailure();
+        assert t.getClass().equals(Throwable.class);
```

Comment:
For the reasons explained above, In this case, I'd suggest adding this import …

```java
import static org.junit.Assert.assertEquals;
```

… and then changing these two lines to:

```java
assertEquals(Throwable.class, t.getClass());
assertEquals("This is a fire drill, not a real fire", t.getMessage());
```

Reply:
@basil 
Thanks for this feedback.  FYI, you can suggest changes (#6 on https://help.github.com/en/articles/reviewing-proposed-changes-in-a-pull-request) to get your intent across quickly and make it easier for other to take your change.